integrations/docker/images/stage-3/chip-build-esp32-qemu/Dockerfile

```diff
@@ -14,7 +14,7 @@ WORKDIR /opt/espressif/qemu
 # Setup QEMU emulator for ESP32 platform
 RUN set -x \
     && git clone --depth 1 -b esp-develop-20210826 https://github.com/espressif/qemu.git ../qemu-src \
-    && ../qemu-src/configure --target-list=xtensa-softmmu --enable-debug --enable-sanitizers --disable-strip --disable-user --disable-capstone --disable-vnc --disable-sdl --disable-gtk \
+    && ../qemu-src/configure --target-list=xtensa-softmmu --enable-debug --disable-strip --disable-user --disable-capstone --disable-vnc --disable-sdl --disable-gtk \
```

Comment:
please explain this change in the summary: *why* is this the correct change.

Since this modifies the image, you have to update chip-build/version as well.

Reply:
When you run QEMU with address sanitizers enabled ,we get following warning at the start
`WARNING: ASan doesn't fully support makecontext/swapcontext functions and may produce false positives in some cases!`.
For some case, this features works fine with Clang toolchain but has issues with GNU.
[Reference issue](https://github.com/espressif/qemu/issues/77).
Also it might consume more CPU or memory resources which might affect the CI .

Reply:
@shripad621git The point is, all that info should be in the commit message.

Reply:
Updated the description.